### PR TITLE
Add percentage labels to volume and brightness indicators

### PR DESCRIPTION
**Pull request by Ferdi265**
This PR adds a textual percentage next to the progress bar in the volume and brightness indicators. This is useful for users who still want to know the exact volume percentage set in practice.

This is off by default and can be enabled with the `show_percentage` boolean setting in the `[server]` block in `CONFIG_HOME/swayosd/config.toml`.

Example config:

```
# ~/.config/swayosd/config.toml
[server]
show_percentage = true
```

### CODE REVIEW

**Review by ErikReider — CHANGES_REQUESTED** [review]
I'd also prefer if the user could disable this. Might wait with this until config support has been added to undo some of the cmdline arg clutter

File: src/server/osd_window.rs
Comment (Ferdi265) [reply src/server/osd_window.rs]: Fixed in 9e12cd5.

I haven't written a lot of Rust lately, so I don't know if there's a better way to wrap the `&str` in an `Option` than to manually construct the `Some()`, but this should do for now.

**Comment by Ferdi265** [comment]
> I'd also prefer if the user could disable this. Might wait with this until config support has been added to undo some of the cmdline arg clutter

Yes, that is my thought as well. I'm completely fine with leaving this PR hanging until config support is added, I might even help with implementing config support if I find time.

**Comment by Ferdi265** [comment]
Rebased on top of #79 and added a configuration option to enable percentage display.

**Comment by ErikReider** [comment]
I'll wait until #79 is merged :)

**Comment by Ferdi265** [comment]
rebased onto the merged config branch

**Comment by rubiin** [comment]
@ErikReider I think this can be merged

**Comment by Ferdi265** [comment]
@ErikReider if you'd like to reduce the number of things in the CLI options I can also remove the CLI option added in this PR (I already have a fixup commit for that locally, I'd just need to push it)

**Comment by ErikReider** [comment]
> @ErikReider if you'd like to reduce the number of things in the CLI options I can also remove the CLI option added in this PR (I already have a fixup commit for that locally, I'd just need to push it)

That would be appreciated! :D

**Comment by Ferdi265** [comment]
> > @ErikReider if you'd like to reduce the number of things in the CLI options I can also remove the CLI option added in this PR (I already have a fixup commit for that locally, I'd just need to push it)
> 
> That would be appreciated! :D

done

**Review by ErikReider — APPROVED** [review]
Thanks for your patience! :)

**Pull request merged**
